modules/ivc_champva/app/models/ivc_champva/vha_10_7959a.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module IvcChampva
+  class VHA107959a
+    include Virtus.model(nullify_blank: true)
+    include Attachments
+
+    attribute :data
+    attr_reader :form_id
+
+    def initialize(data)
+      @data = data
+      @uuid = SecureRandom.uuid
+      @form_id = 'vha_10_7959a'
+    end
+
+    # rubocop:disable Lint/Syntax
+    def metadata
+      {
+        'veteranFirstName' => @data.dig('veteran', 'full_name', 'first'),
+        'veteranLastName' => @data.dig('veteran', 'full_name', 'last'),
+        'zipCode' => @data.dig('veteran', 'address', 'postal_code'),
+        'source' => 'VA Platform Digital Forms',
+        'docType' => @data['form_number'],
+        'businessLine' => 'CMP',
+        'country' => @data.dig('applicant_address', 'country') || 'USA',
```

Comment:
Just for readability try to keep these matching across the models. `country` is under `zipCode` nitpick

Reply:
No. Not doing it